enrollment-server/src/main/java/com/wultra/app/docverify/zenid/provider/ZenidDocumentVerificationProvider.java

```diff
@@ -323,6 +325,40 @@ public void cleanupDocuments(OwnerId id, List<String> uploadIds) throws Document
         return errors;
     }
 
+    @Override
+    public VerificationSdkInfo initVerificationSdk(OwnerId id, Map<String, String> initAttributes) throws DocumentVerificationException {
+        Preconditions.checkArgument(initAttributes.containsKey(ZenidConst.SDK_INIT_TOKEN), "Missing initialization token for ZenID SDK");
+        String token = initAttributes.get(ZenidConst.SDK_INIT_TOKEN);
+
+        ResponseEntity<ZenidWebInitSdkResponse> responseEntity;
+        try {
+            responseEntity = zenidApiService.initSdk(token);
+        } catch (RestClientException e) {
+            logger.warn("Failed REST call to init ZenID SDK, " + id, e);
```

Comment:
Why don't we write `...init ZenID SKD, {}", id, e`?

Reply:
Because I want also the logged exception with the message.

The implementation of logback does not seem to try to detect it from multiple arg call
```
    public void warn(String format, Object arg1, Object arg2) {
        this.filterAndLog_2(FQCN, (Marker)null, Level.WARN, format, arg1, arg2, (Throwable)null);
    }
```

Reply:
Hmm, I see, it is strange to me, because there is slf4j facade and developers should not care (much) about underlying implementation. Ok, let it be, but I will double check the logback + slf4j.